src/Reflex/Process.hs

```diff
@@ -34,10 +39,15 @@ data ProcessConfig t i = ProcessConfig
   -- ^ "stdin" input to be fed to the process
   , _processConfig_signal :: Event t P.Signal
   -- ^ Signals to send to the process
+  , _processConfig_createProcess
+    :: P.CreateProcess -> IO (Maybe Handle, Maybe Handle, Maybe Handle, ProcessHandle)
```

Comment:
Let's move the `error` call into this default so that this type signature is more precise.
```suggestion
    :: P.CreateProcess -> IO (Handle, Handle, Handle, ProcessHandle)
```

Reply:
Actually I'm having second thoughts about this because of how we use `createProcess` below.